Cargo.lock

```diff
@@ -38,18 +38,18 @@ dependencies = [
 
 [[package]]
 name = "addr2line"
-version = "0.22.0"
+version = "0.24.2"
```

Comment:
There's 1000 lines of diff in this file. Did you run `cargo update` without `-p` or is this all a consequence of bumping the hyper requirement to `0.14.31`?

Reply:
good catch, I probably did it without `-p`